.github/aws-oidc-provider/gha-aws-oidc-provider-template.yaml

```diff
@@ -0,0 +1,290 @@
+Transform: AWS::Serverless-2016-10-31
```

Comment:
Maybe not for this change ... but I'm wondering about having a post-merge action that deploys this template if it changes 🤷🏻